### DISCUSSION TITLE
Reporting success and a big Thanks

### DISCUSSION THREAD

**Discussion by marcuslupinus** [Builty/TonexOneController#26] [General]
Hi, I am new to the discussion but have been following what others had to say.

First I want to report that I had success uploading the firmware (1.0.3.2 server) to a WaveShare ESP32-S3-Zero.  I tested sending midi program change from my android cell using an app called MidiCommander and it worked ok.  I also tested buttons on gpio 4 and 6 and it worked too.  I hooked a 5v power adapter to the 5v and gnd pins.

A HUGE thanks to Builty for that very nice project for us ToneX One owners.  You are really awesome.

One general question: did anyone noticed that the board is running a bit hot ?

**Comment by Builty** [comment]
Thanks!

In terms of temperature, they do run warm. I measure my Zero at around 35 - 40 degrees.
The maximum die temp is around 100 degrees, so I don't believe its of any concern.

There is the option to run the cores at a lower frequency to reduce heat. They are running at 240 MHz now, but can be underclocked to 160 MHz. I'll keep any eye on this one.
